### ISSUE TITLE
ChunkLoadError for custom VegaFusion Jupyter widget

### ISSUE THREAD

**Issue by jonmmease** [widgetti/solara#142]
Congrats on the release of Solara!

I'm a maintainer of the [VegaFusion](https://github.com/hex-inc/vegafusion) library, which includes a custom Jupyter widget. In trying this widget with Solara, I hit an error that I wanted to let you know about.  Here's the repro:

In a fresh environemnt, install vegafusion-jupyter, solara, vega-datasets, and altair 5

```
pip install "vegafusion-jupyter[embed]" solara==1.16.0 vega-datasets altair==5.0.1
```

Create  `try_solara.py` Python file:

```python
from vega_datasets import data
import altair as alt
import vegafusion as vf

source = alt.UrlData(
    data.flights_2k.url,
    format={'parse': {'date': 'date'}}
)

brush = alt.selection_interval(encodings=['x'])

# Define the base chart, with the common parts of the
# background and highlights
base = alt.Chart(width=160, height=130).mark_bar().encode(
    x=alt.X(alt.repeat('column')).bin(maxbins=20),
    y='count()'
)

# gray background with selection
background = base.encode(
    color=alt.value('#ddd')
).add_params(brush)

# blue highlights on the transformed data
highlight = base.transform_filter(brush)

# layer the two charts & repeat
chart = alt.layer(
    background,
    highlight,
    data=source
).transform_calculate(
    "time",
    "hours(datum.date)"
).repeat(column=["distance", "delay", "time"])

chart_widget = vf.jupyter.VegaFusionWidget(chart)
```

Launch solara

```
solara run try_solara.py:chart_widget
```

A browser opens on http://localhost:8765/ with this error:

```
Failed to create view for 'VegaFusionView' from module 'vegafusion-jupyter' with model 'VegaFusionModel' from module 'vegafusion-jupyter'
ChunkLoadError: Loading chunk 876 failed.
(error: http://localhost:8765/nullnbextensions/vegafusion-jupyter/876.index.js)
    at xt.f.j (http://localhost:8765/static/nbextensions/vegafusion-jupyter/index.js:1:524267)
    at http://localhost:8765/static/nbextensions/vegafusion-jupyter/index.js:1:522143
    at Array.reduce ()
    at xt.e (http://localhost:8765/static/nbextensions/vegafusion-jupyter/index.js:1:522120)
    at rt.render (http://localhost:8765/static/nbextensions/vegafusion-jupyter/index.js:1:515768)
    at http://localhost:8765/_solara/cdn/@widgetti/solara-vuetify-app@4.0.0/dist/solara-vuetify-app8.min.js:2:4797889
```

The expected result is a chart that looks like this, with cross filtering enabled:

![Screenshot 2023-06-04 at 5 42 06 PM](https://github.com/widgetti/solara/assets/15064365/eb307230-94d2-45ef-b1b7-57e38faaaca8)

This widget does work successfully in Voila when pulling the widget from CDN, and it works in Visual Studio Code and Colab which also dynamically download the widget from a CDN location.

Thanks!

Edit: The widget implementation is here: https://github.com/hex-inc/vegafusion/tree/main/python/vegafusion-jupyter

**Comment by maartenbreddels** [comment]
Hi Jon,

Thank you! Great to see you showing interest in solara. VegaFusion was on my list of widgets to try out, so happy you did :)
This is now fixed with https://github.com/widgetti/solara/commit/42af250aae26c3f4fc3571b8b663412d47f091cc and released as 1.16.1

Regards,

Maarten

**Comment by jonmmease** [comment]
Thanks for the lightning fast fix @maartenbreddels. For context, this came up in the discussion over in https://github.com/hex-inc/vegafusion/issues/334.  As I said over there, with this fix I think Solara is currently the best solution for using VegaFusion outside of a notebook environment. Great work!
